index.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+
+    <title>Document</title>
+</head>
+
+<body>
+    <header id="toppart">
+        <nav class="toolbar">
+            <a href="">
+                <img src="images/My Logo.svg" width="90" height="90">
+            </a>
+            <a href=""><img src="images/Hamburger button.svg" alt="three lines" width="24" height="24"></a>
+        </nav>
+
+    </header>
+
+    <section id="Introduction">
+        <h1>I'm Lethabo Glad to see you!</h1>
+        <div class="mid">
+            <p>I'm a software developer! I can help you build a product, feature or website Look through some fo my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        </div>
+        <hx class="center">LET'S CONNECT</hx>
+        <ul class="Socials">
+            <li>
+                <a href="#"><img src="images/Twitter.svg" alt="Twitter"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Linkedin.svg" alt="Linkedin"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Medium.svg" alt="Medium"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/GitHub.svg" alt="GitHub"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Angel list.svg" alt="ANGEL LIST"></a>
+            </li>
+        </ul>
+    </section>
+    <section class="greybackground">
+    <section id="wrap">
+        <div class="whitebox">
+            <div class="imageclass">
+                <img src="images/orange ting.svg" height="250" />
+            </div>
+            <div class="description">
+                <h2 class="heading">Tonic</h2>
+
+                <div>
+                    <ul class="list_align">
+                        <li class="bold">CANOPY</li><img src="images/Counter.svg" />
+                        <li>Back End Dev</li><img src="images/Counter.svg" />
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads;no accounts or sign-ups required.
+                </p>
+                <ul class="tags">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>Javascript</li>
+                </ul>
+                <button type="button" class="bluebutton">See More</button>
```

Comment:
- [ ] Your design is really dazzling, you just need small 🤌 changes like this. Please, kindly ensure that the button properties are all applied. the hover and click properties seem the same in your design. But it is different in the Figma design.

- Figma design
<img width="392" alt="right" src="https://user-images.githubusercontent.com/41163571/227228961-c8f83b8d-0f27-4323-936c-3d78ab332b95.png">

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/227229234-9e76e64e-4c01-4aa4-ac4b-71d16db92f6e.gif)

And please kindly
- Make the test in the button a bit more bold to match the design
- Please, also reduce the `border-radius` a bit. 😀